File: CPP/RathodTej/1768. Merge Strings Alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string ans;
        for(int i = 0, j = 0; i < word1.size() || j < word2.size();)
        {
            if(i < word1.size())
            {
                ans +=word1[i];
                i++;
            }
             if(j < word2.size())
            {
                ans +=word2[j]; 
                j++;
            }
        }
        return ans;
    }
};

File: Python/1768. Merge Strings Alternately.py
class Solution:
    def mergeAlternately(self, word1, word2):
        i=0
        res = []
        while i < len(word1) and i < len(word2):
            res.append(word1[i])
            res.append(word2[i])
            i+=1
            #j+=1
        res.append(word1[i:])
        res.append(word2[i:])    
        return "".join(res) 
           
      
